javacutil/src/main/java/org/checkerframework/javacutil/PluginUtil.java

```diff
@@ -26,7 +26,10 @@
 /**
  * This file contains basic utility functions that should be reused to create a command-line call to
  * {@code CheckerMain}.
+ *
+ * @deprecated Renamed to {@link SystemUtil}
```

Comment:
Not everything from here will be in `SystemUtil`. Should the document clarify that some methods will completely go away?